### DISCUSSION TITLE
create student acompte

### DISCUSSION THREAD

**Discussion by KalouiSanaa** [community/community#52705] [GitHub Education]
Good morning,
I'm trying to create a student account with my openclassroom school certificate but I can't. Can I send it to you directly?
[THANKS

**Comment by samus-aran** [comment]
Hi @KalouiSanaa  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants.
